sysdig/resource_sysdig_monitor_notification_channel_opsgenie_test.go

```diff
@@ -52,3 +60,15 @@ resource "sysdig_monitor_notification_channel_opsgenie" "sample-opsgenie" {
 	notify_when_resolved = false
 }`, name)
 }
+
+func monitorNotificationChannelOpsGenieWithNameAndRegion(name string) string {
```

Comment:
we need to update the secure brother too
https://github.com/sysdiglabs/terraform-provider-sysdig/blob/master/sysdig/resource_sysdig_secure_notification_channel_opsgenie_test.go

Reply:
you are absolutely right, I added the same test also there now.